includes/admin/settings/class-wcmp-settings-data.php

```diff
@@ -485,6 +490,20 @@ private function get_section_general_general(): array
         ];
     }
 
+    /**
+     * @return array
+     */
+    private function get_section_general_widget(): array
```

Comment:
doe maar gewoon camelcase, ookal is dat in de rest van het bestand ook (nog) niet zo